packages/archway-proto/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
The root folder already contains a `.editorconfig` file. You can delete this one.